x-pack/plugin/src/test/resources/rest-api-spec/test/ml/jobs_crud.yml

```diff
@@ -1031,100 +898,6 @@
       ml.close_job:
         job_id: jobs-crud-some-missing-job-i-made-up
 
----
```

Comment:
These "create with closed index" and "create with previously existing docs" tests are now covered in: `org.elasticsearch.xpack.ml.integration.AnomalyJobCRUDIT`